dev/com.ibm.ws.microprofile.reactive.streams.operators/src/com/ibm/ws/microprofile/reactive/streams/WASReactiveStreamsEngineImpl.java

```diff
@@ -0,0 +1,41 @@
+package com.ibm.ws.microprofile.reactive.streams;
+
+import java.util.concurrent.CompletionStage;
+import java.util.concurrent.ForkJoinPool;
+
+import org.eclipse.microprofile.reactive.streams.CompletionSubscriber;
+import org.eclipse.microprofile.reactive.streams.spi.Graph;
+import org.eclipse.microprofile.reactive.streams.spi.ReactiveStreamsEngine;
+import org.eclipse.microprofile.reactive.streams.spi.UnsupportedStageException;
+import org.osgi.service.component.annotations.Component;
+import org.osgi.service.component.annotations.ConfigurationPolicy;
+import org.reactivestreams.Processor;
+import org.reactivestreams.Publisher;
+
+import com.lightbend.microprofile.reactive.streams.zerodep.BuiltGraph;
+import com.lightbend.microprofile.reactive.streams.zerodep.ReactiveStreamsEngineImpl;
+
+@Component(name = "com.ibm.ws.microprofile.reactive.streams.ReactiveStreamsEngineImpl", service = { ReactiveStreamsEngine.class }, configurationPolicy = ConfigurationPolicy.IGNORE, immediate = true)
```

Comment:
Although we suggested earlier that you didn't need a DS, this suggest you do since you need to get a reference to whatever the EE equivalent of a ForkJoinPool is?